video/beta/src/test/java/com/example/video/DetectIT.java

```diff
@@ -57,4 +65,64 @@ public void testSpeechTranscription() throws Exception {
 
     assertThat(got).contains("cultural");
   }
+
+  @Test
+  public void testTrackObjects() throws Exception {
+    VideoAnnotationResults result = TrackObjects.trackObjects("resources/cat.mp4");
+
+    boolean textExists = false;
+    for (ObjectTrackingAnnotation objectTrackingAnnotation : result.getObjectAnnotationsList()) {
+      if (objectTrackingAnnotation.getEntity().getDescription().toUpperCase().contains("CAT")) {
```

Comment:
```java
for (ObjectTrackingAnnotation objectTrackingAnnotation : result.getObjectAnnotationsList()) {
    String desc = objectTrackingAnnotation.getEntity().getDescription().toUpperCase();
    assertThat(desc, CoreMatchers.containsString("Contains"));
}
```

Reply:
I think I looked at this wrong last night - is this trying to make sure that at least one of `POSSIBLE_TEXTS` was added as an annotation? If so, we should add a break after `textExists = true` since we only need one.

Reply:
Yea, we were trying to make the test less flaky. 
Will add the break.